README.md

```diff
@@ -1,64 +1,47 @@
-# Docker + Nginx + ReactJS + Django + PostgreSQL
+# Project ThunderCAT
 
-This project serves as an example of a deployment frontend (ReactJS) and backend (Django) using docker and nginx.
+This application is a collaboration between Code for Canada and the Public Service Commission of Canada.
 
-## How to run
+This project serves as an example of a deployment front-end (ReactJS) and back-end (Django with PostgreSQL DB) using docker and nginx.
 
-Make sure you have [docker](https://docs.docker.com/install/) and [docker-compose](https://docs.docker.com/compose/install/) installed.
+## Quick start
 
- Run:
+1. Follow our [setup guide](./SETUP.md) to configure your workstation.
+2. Clone the repository using [git](./doc/contributing-with-git.md)
+3. `docker-compose up` to build the application if you have not done so already
+4. Open your browser to [http://localhost:80/](http://localhost:80/).
 
-```shell
-docker-compose up
-```
+To see a list of all build commands, run `docker` from the repository root or see our [notes-on-docker](./docs/notes-on-docker.md).
 
-Then, in the browser, go to localhost:80 and you should see the application with a response from the backend.
+## How to help
 
-When killing the server, be sure to cleanup your containers.
+Interested in what we're working? [Learn more about us and our project](https://medium.com/code-for-canada/inboxing-clever-db6a334dda7d).
+Want to contribute? Please email caley@codefor.ca.
 
-```shell
-docker-compose down
-```
+## What's in this repo?
+Here's a quick overview of the major landmarks:
 
-## How to tests
-This script will run both backend and frontend tests
+### [backend](./backend)
 
-In gitbash:
-```shell
-./run-tests.sh
-```
-In powershell:
-```shell
-.\run-tests.sh
-```
+The server for our CAT platform, a [Django](https://www.djangoproject.com/) application responsible for:
 
-Note: will need to press [Ctrl]+[C] after npm tests finish
+* not much yet, but it will hold our APIs
 
+### [frontend](./frontend)
 
-## Notes on docker
+The [ReactJS](https://reactjs.org/) application that users will see. It users Airbnb Styleguide. It will get built into a static package that we serve through nginx once we deploy it.
```

Comment:
Do you mean 'It **uses** Airbnb styleguide'?

Reply:
Updated.